README.md

```diff
@@ -268,6 +268,20 @@ val usersSource = Clump.source(fetch _)(_.id)
 val userClump = usersSource.get(id)
 ```
 
+### Map with Success and Failure ###
+
+Sometimes sources can mark individual entries as failed, even though the entire fetch function succeeded. For these sources, a function can be provided that maps from id to a Try object that can be marked as either Success or Failure.
+
+```scala
+def fetch(ids: List[Int]): Future[Map[Int, Try[User]]] = ...
+
+val usersSource = Clump.source(fetch _)
```

Comment:
You're right, I think I fixed it in the docs PR though, will double check.

On Mon, Nov 2, 2015, 16:17 William Boxhall <notifications@github.com> wrote:

> In README.md
> <https://github.com/getclump/clump/pull/109#discussion_r43702956>:
>
> > @@ -268,6 +268,20 @@ val usersSource = Clump.source(fetch _)(_.id)
> >  val userClump = usersSource.get(id)
> >  ```
> >
> > +### Map with Success and Failure ###
> > +
> > +Sometimes sources can mark individual entries as failed, even though the entire fetch function succeeded. For these sources, a function can be provided that maps from id to a Try object that can be marked as either Success or Failure.
> > +
> > +```scala
> > +def fetch(ids: List[Int]): Future[Map[Int, Try[User]]] = ...
> > +
> > +val usersSource = Clump.source(fetch _)
>
> should this be sourceTry in the docs here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getclump/clump/pull/109/files#r43702956>.
>